courses/templates/courses/partials/after_the_course_body.html

```diff
@@ -0,0 +1,213 @@
+{% load staticfiles %}
+
+<div class="discover-uni-container">
+    <div class="after-course__lead">
+        <h3 class="after-course__lead-title">
+            What you need to know about earnings data
+        </h3>
+
+        <h4 class="after-course__lead-heading">
+            Labour markets change
+        </h4>
+
+        <p>
+            Earnings data from past graduates can only give an indication of what the labour market and earnings might be like when you graduate.
+        </p>
+
+        <h4 class="after-course__lead-heading">
+            Salaries vary across regions in the UK
+        </h4>
+
+        <p>
+            For example a job in London may be paid more highly than a similar job elsewhere. If a lot of graduates from a course go on to work in a higher pay region, the salary from that institution may be higher – this doesn’t necessarily reflect on the course they studied.
+        </p>
+
+        <h4 class="after-course__lead-heading">
+            There are lots of factors that affect graduates’ earnings
+        </h4>
+
+        <p>
+            Not just the course or institution.
+        </p>
+    </div>
+
+    <div class="after-course__block">
+        <div class="after-course__data-point">
+            <h3 class="after-course__stats-heading">
+                Earnings 6 months after the course
+            </h3>
+
+            <p class="after-course__figure">
+                £{{ course_details.salary_stats.mean }}
+            </p>
+
+            <p class="after-course__figure-explanation">
+                is the average earnings for all courses in
+                {% if course_details.salary_stats.unavailable_reason %}
+                    {{ course_details.salary_stats.subject_english_name }}
+                {% else %}
+                    {{ course_details.english_title }}
+                {% endif %}
+                after 6 months from the
+                <strong>{{course_details.institution.pub_ukprn_name}}</strong>
+            </p>
+
+            <p>
+                The typical salary range is:
```

Comment:
The dataset API now returns sector level salary stats; these can now be used in this p tag. The salary json will look something like this now:
```
salary: [
  {
    "aggregation_level": 14,
     "higher_quartile": 24000,
     "lower_quartile": 22000,
     "median": 22000,
     "number_of_graduates": 65,
     "response_rate": 91,
     "sector_higher_quartile": 26000,
     "sector_lower_quartile": 22000,
     "sector_median": 22000
  }
]
```